bundles/org.openhab.binding.vallox/src/main/java/org/openhab/binding/vallox/internal/se/mapper/ChannelMapper.java

```diff
@@ -0,0 +1,193 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.vallox.internal.se.mapper;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link ChannelMapper} maps all channel information together.
+ *
+ * @author Miika Jukka - Initial contribution
+ */
+@NonNullByDefault
+public class ChannelMapper {
+
+    private static final String CHANNEL_GROUP_FAN = "fanControl#";
+    private static final String CHANNEL_GROUP_TEMPERATURE = "temperature#";
+    private static final String CHANNEL_GROUP_EFFICIENCY = "efficiency#";
+    private static final String CHANNEL_GROUP_SETTINGS = "setting#";
+    private static final String CHANNEL_GROUP_STATUS = "status#";
+    private static final String CHANNEL_GROUP_MAINTENANCE = "maintenance#";
+    private static final String CHANNEL_GROUP_ALARM = "alarm#";
+
+    @SuppressWarnings("serial")
+    private static final Map<String, ValloxChannel> VALLOXSE = Collections
```

Comment:
Is there anyway you can make this an enum instead?

Reply:
Any specific reason to do so?

Reply:
I'll change to enum as you suggested.